Faq style guide template (#11680)
cloudflare/cloudflare-docs: content/style-guide/documentation-content-strategy/content-types/faq.md

Make sure each question includes the entire text of the question.

> Make sure the answer includes the entire answer, as well as a direct response to the question (if the question is phrased in a Yes/No manner).

Reviewer: ```suggestion
Make sure the answer includes the entire answer, and a direct response to the question (if the question is phrased in a Yes/No manner).
```
